### PR TITLE
Use forward in prediction and add tf.ConfigProto

### PR DESCRIPTION
**Pull request by yangw1234**

### CODE REVIEW

File: pyzoo/zoo/pipeline/api/net/tfnet.py [hunk 1]
```diff
@@ -107,30 +125,48 @@ def predict(self, x, batch_per_thread=1, distributed=True):
...
         else:
-            if isinstance(x, np.ndarray) or isinstance(x, list):
-                results = callBigDlFunc(self.bigdl_type, "zooPredict",
-                                        self.value,
-                                        self._to_jtensors(x),
-                                        batch_per_thread)
-                return [Layer.convert_output(result) for result in results]
-            else:
-                raise TypeError("Unsupported prediction data type: %s" % type(x))
+            self.forward(x)
```
Comment (jason-dai) [inline pyzoo/zoo/pipeline/api/net/tfnet.py]: You need to deal with `batch_per_thread` here

**Comment by jason-dai** [comment]
Please add more descriptions wrt `tf.ConfigProto`, esp. how users might use it.

File: pyzoo/zoo/pipeline/api/net/tfnet.py (hunk 1 above)
Comment (yangw1234) [reply pyzoo/zoo/pipeline/api/net/tfnet.py]: fixed

**Comment by yangw1234** [comment]
http://172.168.2.101:8080/job/ZOO-PR-Validation/2407/
jenkins pass

**Pull request merged**